### ISSUE TITLE
Add a footer to the website

### ISSUE THREAD

**Issue by imsahil007** [sajag1999/Website-Blocker-Project#22]
**Task**

Add a footer for the front-end page our to be website.

**Implementation**

The footer should be in colour contrast with the navbar.
It should be responsive and should work on small view-port devices as well.

**Procedure**

Claim this issue: Comment below that you want to work on this issue
Write a relevant commit message
Create a PR

All website discussion to be done  under #7

**Comment by kmrakash** [comment]
can i work on this issue ?
please assign and add SCI LABEL so that i can move forward

**Comment by kmrakash** [comment]
what should i add in footer section ?
